build-release.sh

```diff
@@ -23,50 +23,68 @@ sign_windows() {
+  ssh "${OSX_HOST}" "while xcrun altool --notarization-info ${request_uuid} -u \"${APPLE_ID}\" -p \"${APPLE_ID_PASSWORD}\" | grep -q Status:\ in\ progress; do echo Waiting on Apple notarization...; sleep 30s; done"
+  if ! ssh "${OSX_HOST}" "xcrun altool --notarization-info ${request_uuid} -u \"${APPLE_ID}\" -p \"${APPLE_ID_PASSWORD}\" | grep -q Status:\ success"; then
+    echo "Notarization failed."
+    notarization_log=$(ssh "${OSX_HOST}" "xcrun altool --notarization-info ${request_uuid} -u \"${APPLE_ID}\" -p \"${APPLE_ID_PASSWORD}\"")
+    echo "${notarization_log}"
+    ssh "${OSX_HOST}" "rm -rf ${osx_tmpdir}"
+    exit 1
+  else
     ssh "${OSX_HOST}" "
-              cd ${osx_tmpdir} && \
-              unzip ${binname}.zip && \
-              codesign --force --timestamp \
-                --options=runtime --entitlements ${entitlements} \
-                -s ${OSX_KEY_ID} -v ${extra_files} ${appname} && \
-              zip -r ${binname}_signed.zip ${appname}"
-
-    request_uuid=$(ssh "${OSX_HOST}" "xcrun altool --notarize-app --primary-bundle-id \"${OSX_BUNDLE_ID}\" --username \"${APPLE_ID}\" --password \"${APPLE_ID_PASSWORD}\" --file ${osx_tmpdir}/${binname}_signed.zip")
-    request_uuid=$(echo ${request_uuid} | sed -e 's/.*RequestUUID = //')
-    ssh "${OSX_HOST}" "while xcrun altool --notarization-info ${request_uuid} -u \"${APPLE_ID}\" -p \"${APPLE_ID_PASSWORD}\" | grep -q Status:\ in\ progress; do echo Waiting on Apple notarization...; sleep 30s; done"
-    if ! ssh "${OSX_HOST}" "xcrun altool --notarization-info ${request_uuid} -u \"${APPLE_ID}\" -p \"${APPLE_ID_PASSWORD}\" | grep -q Status:\ success"; then
-      echo "Notarization failed."
-      notarization_log=$(ssh "${OSX_HOST}" "xcrun altool --notarization-info ${request_uuid} -u \"${APPLE_ID}\" -p \"${APPLE_ID_PASSWORD}\"")
-      echo "${notarization_log}"
-      ssh "${OSX_HOST}" "rm -rf ${osx_tmpdir}"
-      exit 1
-    else
-      ssh "${OSX_HOST}" "
-              cd ${osx_tmpdir} && \
-              xcrun stapler staple ${appname} && \
-              zip -r ${binname}_stapled.zip ${appname}"
-      scp "${OSX_HOST}:${osx_tmpdir}/${binname}_stapled.zip" ${reldir}/${binname}.zip
-      ssh "${OSX_HOST}" "rm -rf ${osx_tmpdir}"
-    fi
+            cd ${osx_tmpdir} && \
+            xcrun stapler staple ${appname} && \
+            zip -r ${binname}_stapled.zip ${appname}"
+    scp "${OSX_HOST}:${osx_tmpdir}/${binname}_stapled.zip" ${reldir}/${binname}.zip
+    ssh "${OSX_HOST}" "rm -rf ${osx_tmpdir}"
+  fi
+}
+
+sign_macos_template() {
+  if [ -z "${OSX_HOST}" ]; then
+    return
   fi
+  osx_tmpdir=$(ssh "${OSX_HOST}" "mktemp -d")
+  reldir="$1"
+
+  scp "${reldir}/osx.zip" "${OSX_HOST}:${osx_tmpdir}"
+  ssh "${OSX_HOST}" "
+            cd ${osx_tmpdir} && \
+            unzip osx.zip && \
+            codesign --force -s - \
+              --options=linker-signed \
+              -s ${OSX_KEY_ID} -v ${extra_files} ${appname} && \
```

Comment:
```suggestion
              -v ${extra_files} ${appname} && \
```

That's real identity, should not be here.

Reply:
Fixed.